Description: Support DEB_BUILD_OPTIONS='noopt nostrip'
 This patch adds support for DEB_BUILD_OPTIONS=noopt and
 DEB_BUILD_OPTIONS=nostrip, which are convenient for debugging.
Author: Michael Gold <michael@bitplane.org>
Bug-Debian: http://bugs.debian.org/737364

Index: higan-099/higan/GNUmakefile
===================================================================
--- higan-099.orig/higan/GNUmakefile	2016-06-10 20:46:19.068002449 +0200
+++ higan-099/higan/GNUmakefile	2016-06-10 20:47:04.476911266 +0200
@@ -4,7 +4,8 @@
 # target := loki
 # console := true
 
-flags += -I. -I.. -O3
+flags += -I. -I..
+flags   += $(if $(filter noopt,$(DEB_BUILD_OPTIONS)),-O0,-O3)
 objects := libco audio video resource
 
 # profile-guided optimization mode
